src/helpers/jwtHelper.js

```diff
@@ -55,11 +55,17 @@ class JWTHelper {
 
   createToken(entity) {
     try {
+      const {  id, email, created_date } = entity;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'created_date' is not in camel case.](https://app.codacy.com/gh/Full-Moon-Castle/werewolf/pullRequest?prid=5345778)